### ISSUE TITLE
Relax requirements on requests

### ISSUE THREAD

**Issue by arnaudlimbourg** [keenlabs/KeenClient-Python#133]
Hello,

Would it be possible to relax the requirements on the requests package. I've been experimenting with `pipenv` which uses` pip-tools` and complains when there is a mismatch in the dependencies:
```
pipdeptree -r -p requests                                                                         
Warning!!! Possibly conflicting dependencies found:
* keen==0.4.0
 - requests [required: >=2.5,<2.11.0, installed: 2.18.4]
------------------------------------------------------------------------
```

I've been using the `keen` library with an up-to-date requests for a while and there does not seem to be any issue (granted it's not a scientific approach).

Would it be okay to have `>=2.5,<3` as a version requirement?

**Comment by masojus** [comment]
I believe this is addressed in PR #132 as part of fixing Issue #129

**Comment by arnaudlimbourg** [comment]
Indeed I had missed that, great!
